common/persistence/pinotVisibilityTripleManager.go

```diff
@@ -452,10 +472,15 @@ func (v *pinotVisibilityTripleManager) GetClosedWorkflowExecution(
 	ctx context.Context,
 	request *GetClosedWorkflowExecutionRequest,
 ) (*GetClosedWorkflowExecutionResponse, error) {
+	earlistTime := int64(0) // this is to get all closed workflow execution
```

Comment:
did you want to hardcode these values?

Reply:
Yes. My understanding for GetClosedWorkflowExecutionRequest is to get all the closed workflow. Am I correct?

Reply:
I don't think so? Like this should be for a time range?

Reply:
Discussed offline, Bowen correctly points out that the request struct doesn't have a lower bound (for whatever reason), so this value is not used here.